crates/notion-core/src/error/details.rs

```diff
@@ -184,26 +186,39 @@ impl fmt::Display for ErrorDetails {
         match self {
             ErrorDetails::BinaryAlreadyInstalled {
                 bin_name,
-                package,
-                version,
+                existing_package,
+                new_package,
             } => write!(
                 f,
-                "Conflict with bin '{}' already installed by '{}' version {}",
-                bin_name, package, version
+                "Executable '{}' is already installed by {}
+
+Remove {} before installing {}",

```

Comment:
For consistency with the other CTAs, this should probably be `Please remove`.

Reply:
Very true, updated.